pkg/tools/tools.go

```diff
@@ -188,13 +188,24 @@ func ToolResultToMap(result any) (map[string]any, error) {
 // LoadAndRegisterCustomTools loads tool configurations from a YAML file
 // and registers them.
 func LoadAndRegisterCustomTools(configPath string) error {
-	if configPath == "" {
-		// Default config path: ~/.config/kubectl-ai/tools.yaml
-		home, err := os.UserHomeDir()
+	pathInfo, err := os.Stat(configPath)
+	if err != nil {
```

Comment:
If the default config paths do not exist, we shouldn't error out. We should only error out if user specified path doesn't exist or we fail to read.

Reply:
In this case, erroring out means a warning in the log.

Reply:
I agree with the idea. Do you mind if I do that as a separate PR?

Reply:
Separate PR is totally fine.